fix a spurious test failure

Timing w/ a TTW test.

https://travis-ci.org/gratipay/gratipay.com/builds/156678522

Here's one:

https://travis-ci.org/gratipay/gratipay.com/builds/156897977#L1077-L1094

```
______________________ Tests.test_owner_can_add_a_member _______________________
self = <test_team_distributing.Tests testMethod=test_owner_can_add_a_member>
    def test_owner_can_add_a_member(self):
        self.sign_in('picard')
        self.visit('/TheEnterprise/distributing/')
        self.css('.lookup-container .query').first.fill('alice')
        time.sleep(0.1)
        self.css('.lookup-container button').first.click()
        time.sleep(0.1)
>       assert [a.text for a in self.css('table.team a')] == ['alice']
E       AssertionError: assert [] == ['alice']
E         Right contains more items, first extra item: 'alice'
E         Use -v to get the full diff
tests/ttw/test_team_distributing.py:26: AssertionError
```

![screen shot 2016-09-01 at 2 58 15 pm](https://cloud.githubusercontent.com/assets/134455/18180360/87554d38-7054-11e6-9787-fc6629c1d564.png)

--- a/tests/ttw/test_team_distributing.py
+++ b/tests/ttw/test_team_distributing.py
@@ -22,7 +22,7 @@ class Tests(BrowserHarness):
         self.css('.lookup-container .query').first.fill('alice')
         time.sleep(0.1)
         self.css('.lookup-container button').first.click()
-        time.sleep(0.1)
+        time.sleep(0.2)
         assert [a.text for a in self.css('table.team a')] == ['alice']
 
 

